javascript/fluent.js

```diff
@@ -173,6 +173,17 @@
 				}
 				input.change();
 			}
-		})
+		});
+
+		/**
+		 * Maintain UI state of the CMS locale switcher when using standard browser navigation buttons.
+		 */
+		$('.cms-edit-form').entwine({
+			redraw: function() {
+				var localeMenuLocale = $.cookie('FluentLocale_CMS')
```

Comment:
This string is soft-coded in https://github.com/tractorcow/silverstripe-fluent/blob/master/_config/fluent.yml#L44

Can you make sure the fluent config includes the cookie name, then soft-coding it here?